src/backend/api/reservations.js

```diff
@@ -0,0 +1,68 @@
+const express = require("express");
+const router = express.Router();
+const knex = require("../database");
+
+// Returns all reservations
+router.get("/", async(request, response) => {
+    try {
+        let queryfromDB = knex("reservation");
+        const result = await queryfromDB.select();
```

Comment:
I suggest using await with knex, select with required data like id, title, etc., and never use select()/(*).